addons/edi/models/edi_document.py

```diff
@@ -84,6 +84,11 @@ def _default_project_id(self):
         default=True, string="Active", help="Display in list views or searches."
     )
 
+    # Optionally enforce filename globs
+    enforce_filename = fields.Boolean(string="Enforce Document Filenames",
```

Comment:
Nit: I think indentation is not correct, try formatting

Reply:
Yeah good spot, I ran black formatting for the whole file and there were quite a few changes